### PR TITLE
Fix Cannot Create Graphql APIs

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
Fix issue https://github.com/wso2/carbon-apimgt/issues/7032

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 1]
```diff
@@ -224,12 +228,77 @@ public String generateAPIDefinition(SwaggerData swaggerData) throws APIManagemen
...
+            }
         }
         return Json.pretty(openAPI);
     }
 
+    /**
+     * Construct openAPI definition for graphQL. Add get and post operations
+     *
+     * @param openAPI OpenAPI
+     */
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: @return is missing in the comment

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 2]
```diff
@@ -224,12 +228,77 @@ public String generateAPIDefinition(SwaggerData swaggerData) throws APIManagemen
...
+    private OpenAPI modifyGraphQLSwagger(OpenAPI openAPI) {
+        SwaggerData.Resource resource = new SwaggerData.Resource();
+        resource.setAuthType("Any");
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: You can use the APIConstants.AUTH_APPLICATION_OR_USER_LEVEL_TOKEN here

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 3]
```diff
@@ -224,12 +228,77 @@ public String generateAPIDefinition(SwaggerData swaggerData) throws APIManagemen
...
+        resource.setPolicy("Unlimited");
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Use APIConstants.DEFAULT_APP_POLICY_UNLIMITED

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 4]
```diff
@@ -224,12 +228,77 @@ public String generateAPIDefinition(SwaggerData swaggerData) throws APIManagemen
...
+        resource.setPath("/*");
+
+        resource.setVerb("GET");
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Please use available constants

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 1 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 2 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 4 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 3 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed

**Pull request merged**